protocol/README.md

```diff
@@ -4,38 +4,49 @@ This document describes the canonical system architecture used to encrypt and de
 
 ## Architecture
 
-![arch](../diagrams/arch.png)
-
-The canonical architecture contains three major components.
+The canonical architecture contains four major components.
 
 * *TDF Client* - Initiates and drives the TDF encryption and decryption workflows. Only component with access to the content (ciphertext or plaintext).
-* *Entity Attribute Service* (EAS) - Responsible for authenticating TDF Clients and vending them signed [Entity Objects](../schema/EntityObject.md), which act as bearer tokens. The source of truth for attribute provenance and association. A single architecture can have many EAS instance running, for instance, for a TDF Client to join attributes across mulitple orginizations.
-* *Key Access Service* (KAS) - Responsible for authorizing and granting TDF Clients access to rewrapped data key material. If authorized, TDF Clients can use this rewrapped data key to decrypt TDF ciphertext.
+* *OpenID Connect (OIDC) Identity Provider (IdP)* - This system could be any OIDC IdP software.  Virtru has chosen Keycloak as our reference implementation IdP.
```

Comment:
The OpenID spec does not specify _how_ a compliant OIDC IdP obtains the custom claim data it inserts into OIDC tokens - and since getting the right claims into the token is essential for our stack (KAS) to work correctly, it makes sense to talk about how we actually accomplish this in a reference implementation.

Reply:
The language choices we've made attempt to define the TDF spec and then our own reference implementation of that spec.  The goal would be to make the protocol less of a purely abstract concept by providing concrete examples and also to describe what a prospective OSS developer would receive when downloading our software and why each component is there.